parsers/BO.py

```diff
@@ -74,64 +66,79 @@ def fetch_production(zone_key='BO', session=None, target_datetime=None, logger=N
             timestamp = now
 
         if target_datetime is not None and hour < 24:
-            timestamp = timestamp.replace(hour=hour-1)
+            timestamp = timestamp.replace(
+                hour=hour - 1, minute=0, second=0, microsecond=0
+            )
+
+        result.append(
+            HourlyProduction(
+                datetime=timestamp.datetime,
+                forecast=forecast,
+                total=total,
+                thermo=thermo,
+                hydro=hydro,
+                wind=wind,
+                solar=solar,
+                bagasse=bagasse,
+            )
+        )
+
+    return result
+
+
+def fetch_production(
+    zone_key="BO", session=None, target_datetime=None, logger=None
+) -> list:
+    """Requests the last known production mix (in MW) of a given country."""
+    payload = []
 
+    for row in fetch_data(
+        session=session, target_datetime=target_datetime, logger=logger
+    ):
+        # NOTE: thermo includes gas + oil mixed, so we set these as unknown for now
+        modes_extracted = [row.hydro, row.solar, row.wind]
```

Comment:
any particular reason we don't want to report bagasse as biomass?

Reply:
Good catch. Will fix. :)